scanner/config/config.go

```diff
@@ -139,6 +139,8 @@ type MatcherConfig struct {
 	// instance at the specified address, instead of the local indexer (when the
 	// indexer is enabled).
 	IndexerAddr string `yaml:"indexer_addr"`
+	// DefinitionsURL sets the URL to pull vulnerability definitions.
+	DefinitionsURL string `yaml:"definitions_url"`
```

Comment:
nit: I was never a huge fan of `definitions` and I think this could be a good opportunity to migrate to `vulnerabilities_url`. Thoughts?

Reply:
Sure. 851f94a8